test/Window/Keyboard.test.cpp

```diff
@@ -14,7 +14,7 @@ TEST_CASE("[Window] sfKeyboard")
         STATIC_CHECK(sfKeyC == static_cast<int>(sf::Keyboard::Key::C));
         STATIC_CHECK(sfKeyD == static_cast<int>(sf::Keyboard::Key::D));
         STATIC_CHECK(sfKeyPause == static_cast<int>(sf::Keyboard::Key::Pause));
-        CHECK(sfKeyCount == sf::Keyboard::KeyCount);
+        STATIC_CHECK(sfKeyCount == sf::Keyboard::KeyCount);
```

Comment:
Keyboard::KeyCount is unsigned int while sfKeyCount is signed wouldn't this cause warnings?

Reply:
If CI doesn't emit any warnings then I'm not worried.